<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-RSES787T" failures="6" tests="69" name="com.runner.Runner" time="193.752" errors="1" timestamp="2023-01-11T02:36:42 EST" skipped="0">
  <testcase classname="com.runner.Runner" name="runScenario" time="2.502"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="2.452"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.372"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.549"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.233"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.335"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="2.217">
    <failure message="Result do not match expected [submission success] but found [ Some Tests failed. Please review code]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Result do not match expected [submission success] but found [ Some Tests failed. Please review code]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at com.stepDefinations.ArraysStepDef.the_user_should_be_presented_with_successful_submission_message(ArraysStepDef.java:133)
at ✽.The user should be presented with successful submission message(file:///C:/Users/valli/Git/DS-ALGO/src/test/resources/features/5arrays.feature:110)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="1.298"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="7.401"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.285"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="2.619">
    <failure message="Result do not match expected [submission success] but found [Error occurred during submission]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Result do not match expected [submission success] but found [Error occurred during submission]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at com.stepDefinations.ArraysStepDef.the_user_should_be_presented_with_successful_submission_message(ArraysStepDef.java:133)
at ✽.The user should be presented with successful submission message(file:///C:/Users/valli/Git/DS-ALGO/src/test/resources/features/5arrays.feature:150)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="2.239"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="2.458">
    <failure message="expected [password should contain atleast eight characters.] but found [password_mismatch:The two password fields didn’t match.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [password should contain atleast eight characters.] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.stepDefinations.RegisterStepDef.user_should_get_password_atleast_eight_char_error_message(RegisterStepDef.java:145)
at ✽.user should get password atleast eight char error message "password should contain atleast eight characters."(file:///C:/Users/valli/Git/DS-ALGO/src/test/resources/features/2register.feature:81)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="2.410">
    <failure message="Result do not match expected [submission success] but found [ Some Tests failed. Please review code]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Result do not match expected [submission success] but found [ Some Tests failed. Please review code]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at com.stepDefinations.ArraysStepDef.the_user_should_be_presented_with_successful_submission_message(ArraysStepDef.java:133)
at ✽.The user should be presented with successful submission message(file:///C:/Users/valli/Git/DS-ALGO/src/test/resources/features/5arrays.feature:130)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="4.658"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.214"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.683"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="1.960"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="1.976"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.132"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.545"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="1.922"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="7.956"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="4.893"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.293"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="0.750"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="2.459">
    <failure message="expected [please enter valid password format] but found [password_mismatch:The two password fields didn’t match.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [please enter valid password format] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.stepDefinations.RegisterStepDef.user_should_get_enter_valid_input_error_message(RegisterStepDef.java:159)
at ✽.user should get enter valid input error message "please enter valid password format"(file:///C:/Users/valli/Git/DS-ALGO/src/test/resources/features/2register.feature:89)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="0.389"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="0.615"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="0.488"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="2.198">
    <failure message="Result do not match expected [submission success] but found [ No tests were collected]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Result do not match expected [submission success] but found [ No tests were collected]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at com.stepDefinations.ArraysStepDef.the_user_should_be_presented_with_successful_submission_message(ArraysStepDef.java:133)
at ✽.The user should be presented with successful submission message(file:///C:/Users/valli/Git/DS-ALGO/src/test/resources/features/5arrays.feature:170)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="1.781"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.236"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="2.420"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="0.615"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.385"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="0.735"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="2.751"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="4.869"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.348"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.255"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="1.820"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="2.383">
    <error message="stale element reference: element is not attached to the page document
  (Session info: chrome=108.0.5359.125)
For documentation on this error, please visit: https://selenium.dev/exceptions/#stale_element_reference
Build info: version: &#039;4.7.1&#039;, revision: &#039;c6795baf1a3&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;17.0.5&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [21cf4880043c974c2e97f0de8dfaf818, getElementText {id=c1c6f73c-8bc0-4b40-9d5a-55d4a5627351}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 108.0.5359.125, chrome: {chromedriverVersion: 108.0.5359.71 (1e0e3868ee06..., userDataDir: C:\Users\valli\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:55971}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:55971/devtoo..., se:cdpVersion: 108.0.5359.125, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on WINDOWS (21cf4880043c974c2e97f0de8dfaf818)] -&gt; id: id_username]
Session ID: 21cf4880043c974c2e97f0de8dfaf818" type="org.openqa.selenium.StaleElementReferenceException">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: stale element reference: element is not attached to the page document
  (Session info: chrome=108.0.5359.125)
For documentation on this error, please visit: https://selenium.dev/exceptions/#stale_element_reference
Build info: version: '4.7.1', revision: 'c6795baf1a3'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.5'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [21cf4880043c974c2e97f0de8dfaf818, getElementText {id=c1c6f73c-8bc0-4b40-9d5a-55d4a5627351}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 108.0.5359.125, chrome: {chromedriverVersion: 108.0.5359.71 (1e0e3868ee06..., userDataDir: C:\Users\valli\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:55971}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:55971/devtoo..., se:cdpVersion: 108.0.5359.125, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on WINDOWS (21cf4880043c974c2e97f0de8dfaf818)] -> id: id_username]
Session ID: 21cf4880043c974c2e97f0de8dfaf818
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:551)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:257)
at org.openqa.selenium.remote.RemoteWebElement.getText(RemoteWebElement.java:207)
at jdk.internal.reflect.GeneratedMethodAccessor116.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:52)
at jdk.proxy2/jdk.proxy2.$Proxy44.getText(Unknown Source)
at com.pageObjects.RegisterPage.validatecompare(RegisterPage.java:155)
at com.stepDefinations.RegisterStepDef.user_should_get_password_not_similar_to_username_error_message(RegisterStepDef.java:171)
at ✽.user should get password not similar to username error message "password can not be similar as username"(file:///C:/Users/valli/Git/DS-ALGO/src/test/resources/features/2register.feature:98)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.317"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.159"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="0.722"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.708"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.277"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.288"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="2.598"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.286"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.296"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.359"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="0.898"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.365"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.328"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.437"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.287"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.848"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.365"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="2.018"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="2.134"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="2.932"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="2.730"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="0.712"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.199"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="1.884"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.303"/>
  <system-out/>
  <testcase classname="com.runner.Runner" name="runScenario" time="3.203"/>
  <system-out/>
</testsuite> <!-- com.runner.Runner -->
